app/abcs/abc_download.py

```
@@ -46,12 +58,65 @@ def check_if_folder_exists(self) -> Path:
+        file_name = Path(file.filename)
+        if file_name.suffix not in ('.csv', '.txt'):
+            ic(f"File {file_name.stem} is not a CSV/TXT file. Skipping...")
+            return
+
+        _csv_task = progress.add_task("T5", f"Extract CSV {file_name.stem}", "Started")
+        zip_ref.extract(file, tmp)
+
+        if file_name.suffix == '.txt':
+            return
+        
+        rename = f"{file_name.stem}_{datetime.now():%Y%m%d}dl"
```

Ruff (0.14.7)

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)